README.md

```diff
@@ -1,742 +1,77 @@
-    .SetResource(Resources.CreateServiceResource("test-zipkin"))
-
-{
-    // ...
-}
-```
-
-#### Traces
-
-```csharp
-var spanExporter = new StackdriverTraceExporter(projectId);
-
-using var tracerFactory = TracerFactory.Create(builder => builder.AddProcessorPipeline(c => c.SetExporter(spanExporter)));
-var tracer = tracerFactory.GetTracer("stackdriver-test");
-
-using (tracer.StartActiveSpan("/getuser", out TelemetrySpan span))
-{
-    span.AddEvent("Processing video.");
-    span.PutHttpMethodAttribute("GET");
-    span.PutHttpHostAttribute("localhost", 8080);
-    span.PutHttpPathAttribute("/resource");
-    span.PutHttpStatusCodeAttribute(200);
-    span.PutHttpUserAgentAttribute("Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0");
-
-    Thread.Sleep(TimeSpan.FromMilliseconds(10));
-}
-```
-
-#### Metrics
-
-```csharp
-var metricExporter = new StackdriverExporter(
-    "YOUR-GOOGLE-PROJECT-ID",
-    Stats.ViewManager);
-metricExporter.Start();
-```
-
-### Using Application Insights exporter in Console App
-
-1. Create [Application Insights][ai-get-started] resource.
-2. Set instrumentation key via telemetry configuration object
-   (`new TelemetryConfiguration("iKey")`). This object may be injected via
-   dependency injection as well.
-3. Instantiate a new instance of `ApplicationInsightsExporter`.
-4. See [sample][ai-sample] for example use.
-
-``` csharp
-using var tracerFactory = TracerFactory.Create(builder => builder
-    .SetResource(Resources.CreateServiceResource("my-service"))
-    .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key"));
-var tracer = tracerFactory.GetTracer("application-insights-test");
-
-using (tracer.StartActiveSpan("incoming request", out var span))
-{
-    span.AddEvent("Start processing video.");
-    Thread.Sleep(TimeSpan.FromMilliseconds(10));
-    span.AddEvent("Finished processing video.");
-}
-```
-### Using Application Insights exporter in Web App
-
-1. Create [Application Insights][ai-get-started] resource.
-2. In Startup class, ConfigureServices method add a call of AddOpenTelemetry 
-3. Configurate Application Insights inside AddOpenTelemetry with call UseApplicationInsights
-4. Set instrumentation key via telemetry configuration object
-    telemetryConfiguration.InstrumentationKey = instrumentationKey;
-5. See [sample][ai-web-sample] for example use.
-
-``` csharp
- services.AddOpenTelemetry((sp, builder) =>
-            {
-                    builder.UseApplicationInsights(telemetryConfiguration =>
-                    {
-                        var instrumentationKey = this.Configuration.GetValue<string>("ApplicationInsights:InstrumentationKey");
-                        telemetryConfiguration.InstrumentationKey = instrumentationKey;
-                    })
-                    .AddRequestAdapter()
-                    .AddDependencyAdapter();
-            });
-```
-
-### Using LightStep exporter

```

Comment:
Can you please move this to the Lightstep exporter readme.md file

Reply:
Sure, moved exporter specific docs to exporter folder.